docker/set_up_ubuntu.sh

```diff
@@ -59,7 +54,7 @@ MYUSER="\$(id -u -n)"
 # Set the prompt to use the username we just figured out plus the container
 # name which is in an environment variable. If there is no CONTAINERNAME,
 # then use the host name.
-PS1="\${MYUSER}@socorro:\w\$ "
+PS1="\${MYUSER}@socorro:\\w\$ "
```

Comment:
shellcheck pointed this out to me, too. The additional `\` escapes the `\` so then `\w` ends up in the `PS1` intentionally.